translate/translate_samples.rb

```diff
@@ -12,46 +12,49 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def translate_text project_id:, text:, language_code:
+def translate_text project_id:
   # [START translate_translate_text]
-  # project_id    = "Your Google Cloud project ID"
-  # text          = "The text you would like to translate"
-  # language_code = "The ISO 639-1 code of language to translate to, eg. 'en'"
+  # TODO(developer): Uncomment this line and replace with your Project ID.
```

Comment:
Shouldn't this code inside the RSpec test?

Reply:
Following sample rubric:

```python
# Example
def test_create_table(
        client, to_delete, capsys):
    dataset_id = temp_dataset_id()
    table_id = temp_table_id()
    dataset_ref = client.dataset(
        dataset_id)
    dataset = bigquery.Dataset(
        dataset_ref)
    client.create_dataset(dataset)
    to_delete.append(dataset)

    # [START bigquery_create_table]
    # TODO(developer): Uncomment these
    #     lines and replace with your
    #     values.
    # from google.cloud import bigquery
    # client = bigquery.Client()
    # dataset_id = 'my_dset'  # replace with your dataset ID
    # table_id = 'my_table'  # replace with your table ID

    dataset = client.dataset(dataset_id)
    table_ref = dataset.table(table_id)
    table = bigquery.Table(table_ref)
    table = client.create_table(table)

    print(
        'Created {}.'.format(
             table.table_id)) 
    # [END bigquery_create_table]

    out, _ = capsys.readouterr()
    assert 'Created my_table.' in out

```

For Ruby, you'd do something like
```ruby
example "translate text" do
  capture do
    # [START translate_translate_text]
    # TODO(developer): Uncomment this line and replace with your Project ID.
    # project_id = "your-project-id"
  
    require "google/cloud/translate"

    translate     = Google::Cloud::Translate.new project: project_id
    text          = "Alice and Bob are kind"
    language_code = "fr"  # The ISO 639-1 code of language to translate to
    translation   = translate.translate text, to: language_code

    puts "Translated '#{text}' to '#{translation.text.inspect}'"
    puts "Original language: #{translation.from} translated to: #{translation.to}"
    # [END translate_translate_text]
  end


  expect(captured_output).to include "Original language: en translated to: fr"
  expect(captured_output).to include(
    %{Translated 'Alice and Bob are kind' to '"Alice et Bob sont gentils"'}
  )
end
```

Reply:
Oh I see what you mean. Yes, I'll refactor to combine `translate_samples.rb` and `translate_samples_spec.rb`. I will keep the quickstart tests separate though because that is a sample application, not a code snippet (which are the only ones that this update applies to).